students/Jean-Baptiste/lessons/lesson01/assignment/assign_01_inventory/electricAppliancesClass.py

```diff
@@ -0,0 +1,24 @@
+"""
+Module for Electric Appliances
+"""
+from inventoryClass import Inventory
+
+class ElectricAppliances(Inventory):
+    def __init__(self, **kwargs):
+        Inventory.__init__(self, **kwargs)
+        self.brand = kwargs["brand"]
+        self.voltage = kwargs["voltage"]
+
+    def return_as_dictionary(self):
+        output_dict = {}
+        output_dict['product_code'] = self.product_code
+        output_dict['description'] = self.description
+        output_dict['brand'] = self.brand
+        output_dict['voltage'] = self.voltage
+        return output_dict
+
+    def check_wattage(self, amp):
+        """
+        Calculate wattage
+        """
+        return self.voltage * amp
```

Comment:
You should get in the habit of adding doc strings in all of your functions. The should include information about the inputs and the outputs:
"""
Calculates wattage (voltage * amperage).
Parameters:
amp: (float)
Returns:
wattage: (float)
"""